src/lib/services/client-metrics/last-seen/last-seen-service.ts

```diff
@@ -42,7 +51,12 @@ export class LastSeenService {
             this.logger.debug(
                 `Updating last seen for ${lastSeenToggles.length} toggles`,
             );
-            await this.featureToggleStore.setLastSeen(lastSeenToggles);
+
+            if (this.config.flagResolver.isEnabled('useLastSeenRefactor')) {
+                await this.lastSeenStore.setLastSeen(lastSeenToggles);
```

Comment:
is it critical that we wait on the last seen being saved or can we return 202 Accepted and process it in the background